playbooks/provisioner/openstack/main.yml

```diff
@@ -4,3 +4,5 @@
   roles:
       - { role: openstack/neutron, action: setup }
       - { role: openstack/nova, action: setup }
+
+- include: plugins.yml
```

Comment:
Please add a "when: provisioner.plugins is defined" and have BMC details be under it.
This is it will give meaning to the include rather than make the user dive into it to try and understand what it does.
Having a "when" statement helps a user understand that in order to run whatever behind that, you need to have a setting of some sort